daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/Engine.scala

```diff
@@ -317,6 +317,7 @@ class Engine(val config: EngineConfig = new EngineConfig(LanguageVersion.StableV
         readAs = readAs,
         validating = validating,
         contractKeyUniqueness = config.contractKeyUniqueness,
+        valueNormalization = config.valueNormalization,
```

Comment:
Do we really need to make that parametric ?

Reply:
as far as I can tell there are two cases where we turn this off:
1. `LargeTransactionTest`
2. `Sandbox`
I don’t see a reason why we should need it in either of them and in combination with the fact that `byKey` flags don’t seem to be normalized I’m a bit worried that this is just resulting in us never running tests that validate on LF < 1.14.

Reply:
We also turn of for the scenario-runner.

Reply:
the scenario runner seems like a reasonable usecase. I think ideally that wouldn’t affect speedy and we would instead call the valueenricher but that can definitely be in a separate PR. Also interested in whether @remyhaemmerle-da agrees that this is preferable.

Reply:
Scenario runner is indeed a reasonable use case. 
I incline toward using the value enricher:
- this would limit the number of parameter in the production code
- this would make the scenario service working similarly as the lerger API in verbose mode.

Reply:
I'll take a stab at using the value enricher for the scenario runner in a subsequent PR.